### ISSUE TITLE
Total volume is incorrect

### ISSUE THREAD

**Issue by just-a-node** [connext/monorepo#2901]
**Describe the bug**

Total volume should show total bridged amounts, currently it is not accurate - way too high for polygon and arbitrum:

![image.png](https://images.zenhubusercontent.com/63331d421176573c6a908c0e/86e595b0-42bf-42e6-b723-74741342054a)

**Comment by pjenviri** [comment]
We directly show the data getting from `https://postgrest.mainnet.connext.ninja/daily_transfer_volume`.

**Comment by just-a-node** [comment]
When aggregating volume, are we taking into account the decimal differences between the assets?

**Comment by just-a-node** [comment]
Could we also add the volume splits per pool here? 

![image.png](https://images.zenhubusercontent.com/63331d421176573c6a908c0e/4b107b9c-e361-42f0-bf64-234f4836e450)

**Comment by preethamr** [comment]
The data is correct for week of 25 Dec.
Later weeks have assets other than USDC, checking to find which of the assets cause this skew.

**Comment by preethamr** [comment]
The data is correct for week of 1 Jan.
The data is correct for week of 8 Jan.

Found the weekly USD totals to be in the ball park of what was seen on chain using the transaction hashes of large transfers. I manually checked all of the large transfers, and used approx ether price during that week for USD conversion

Much of the volume came from:
https://amarok.connextscan.io/address/0x29a519e21d6a97cdb82270b69c98bac6426cdcf9
as expected.

The decimals used by CH seem correct. The only asset who adopted asset has different decimals than cannonical is USDC on BSC. Where USDC has 18 decimals.

Nothing to fix.
